src/main/resources/hudson/plugins/git/UserMergeOptions/config.jelly

```diff
@@ -13,4 +13,7 @@
   <f:entry title="Merge strategy:" field="mergeStrategy">
       <f:select />
   </f:entry>
-</j:jelly>
\ No newline at end of file
+  <f:entry title="Fast Forward Mode:" field="fastForwardMode">
+    <f:select />
```

Comment:
I followed the way the merge strategy was coded because I do not know better. I will have to find examples of how to do this with the enum. However, this would bring two different styles of implementing selections unless I also change the merge startegy, wouldn't it?